### ISSUE TITLE
Glitched Album Cover For Some Entries

### ISSUE THREAD

**Issue by Robby9705** [OxygenCobalt/Auxio#124]
I Don't If This Is A Bug Or Not But For Some Reason Instead Of Showing The Original Album Cover, Auxio Shows Some Generic Image Of Earth (Screenshared Off Of A Zoom Call?) I've Uploaded The Image Here.

This Doesn't Happen To All Of Music But Only Some Specific Entries Seem To Be Suffering From This Bug. 
![IMG_20220429_133122](https://user-images.githubusercontent.com/73337652/165906783-8b53af83-c959-4313-9d28-77f12357d1a0.jpg)

**Comment by Robby9705** [comment]
Yep It's Enabled 
![IMG_20220429_205623](https://user-images.githubusercontent.com/73337652/165975672-fd0ae78c-a0d5-4eb4-aabd-5e2afcea68ae.jpg)

**Comment by OxygenCobalt** [comment]
@Robby9705 That's odd. Can you answer two questions for me?

1. Do the affected files have cover art on them? If so, what file format are those files? (MP3, FLAC, etc.)
2. What phone model are you using?

If I were to guess what's happening, it would be:
- Your files have no cover art that Auxio can extract manually.
- Auxio defaults to MediaStore in an attempt to fix the quality.
- MediaStore has no cover for the file either, so it defaults to the image database for some reason. In this case, it chooses a generic earth image (perhaps because the ID matches???).

**Comment by Robby9705** [comment]
No The Affected Files Don't Have A Cover Art But Some Of Those Files Have An Image Of A Track Record Or Something (That I've Uploaded Here)

As For My Model And Android Version, It's Redmi K2P Pro And Android 11

![IMG_20220430_234950](https://user-images.githubusercontent.com/73337652/166117714-824404e0-29a5-47bd-bda1-cad44fb7ce15.jpg)

As You Can See The First Has A Different Cover Art And The Second One Has That Earth Image (Both Of The Files Don't Have A Cover Art Originally) I Guess Auxio Should Use The First Cover Art For Every File That Doesn't Have A Cover Art By Itself

**Comment by OxygenCobalt** [comment]
Another song applying a default cover does not necessarily disprove it, as the MediaStore image may not actually point to a corresponding image within the image database.

Sadly, it seems impossible for me to detect if cover art actually came from the right database, so I doubt I can fix this. This is completely a problem on Redmi's end (Redmi, Oppo, Huawei, and Samsung are notorious for botching basic android functionality as to railroad you into their built-in apps).

**Comment by rhynec** [comment]
@Robby9705 @OxygenCobalt 

I think this issue is caused by images being cached in the .thumbnails folder inside the Music directory. I fixed the issued by manually deleting the .thumbnails folder and then restarting my phone.

I think that on newer versions of Android, MediaStore attempts to retrieve artwork from external images included inside an album's folder. Though, it seems that in this case the retrieval was too aggressive. I only ran into the issue on my phone (Moto G Fast, basically vanilla Android 11) because I accidentally included an image alongside an album with no artwork, which refused to go away even after deleting the picture and restarting the phone numerous times.
